contracts/LpSugar.vy

```diff
@@ -310,9 +310,9 @@ def forSwaps(_limit: uint256, _offset: uint256) -> DynArray[SwapLp, MAX_POOLS]:
       pool_fee: uint256 = 0
 
       is_cl_pool: bool = False
-      is_stable: bool = factory.getPool(token0, token1, 0) == pool_addr
```

Comment:
Let's add a comment here explaining the flow based on:
https://github.com/velodrome-finance/contracts/blob/v2-optimism/contracts/interfaces/factories/IPoolFactory.sol#L41-L46